tests/model/issue.model.spec.ts

```diff
@@ -1,143 +1,57 @@
+import * as moment from 'moment';
 import { Issue } from '../../src/app/core/models/issue.model';
-import { Phase } from '../../src/app/core/models/phase.model';
-import { TesterResponse } from '../../src/app/core/models/tester-response.model';
-import { TEAM_4 } from '../constants/data.constants';
-
+import { Milestone } from '../../src/app/core/models/milestone.model';
 import { ISSUE_WITH_ASSIGNEES, ISSUE_WITH_EMPTY_DESCRIPTION } from '../constants/githubissue.constants';
 
 describe('Issue model class', () => {
   describe('.createPhaseBugReportIssue(githubIssue)', () => {
     it('correctly creates a bug reporting issue that has an empty description', async () => {
       const issue = Issue.createPhaseBugReportingIssue(ISSUE_WITH_EMPTY_DESCRIPTION);
-      expect(issue.title).toBe('App starts to lag when given large amount of input');
-      expect(issue.description).toBe('No details provided by bug reporter.');
-      expect(issue.severity).toBe('Medium');
-      expect(issue.type).toBe('FunctionalityBug');
-    });
-  });
-  describe('.createPhaseTeamResponseIssue(githubIssue, githubComment)', () => {
-    it('correctly creates a team response issue that has an empty team response', async () => {
-      const dummyTeam = null;
-      const issue = Issue.createPhaseTeamResponseIssue(ISSUE_WITH_ASSIGNEES, dummyTeam);
-      expect(issue.title).toBe('Screen freezes');
-      expect(issue.teamResponse).toBe('No details provided by team.');
-      expect(issue.severity).toBe('Medium');
-      expect(issue.assignees).toContain('anubh-v');
-      expect(issue.assignees.length).toBe(1);
+      expect(issue.globalId).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.id);
+      expect(issue.id).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.number);
+      expect(issue.created_at).toEqual(moment(ISSUE_WITH_EMPTY_DESCRIPTION.created_at).format('lll'));
+      expect(issue.title).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.title);
+      expect(issue.description).toEqual('No details provided by bug reporter.');
+      expect(issue.updated_at).toEqual(moment(ISSUE_WITH_EMPTY_DESCRIPTION.updated_at).format('lll'));
+      expect(issue.closed_at).toEqual(moment(ISSUE_WITH_EMPTY_DESCRIPTION.closed_at).format('lll'));
+      expect(issue.milestone).toEqual(
+        ISSUE_WITH_EMPTY_DESCRIPTION.milestone ? new Milestone(ISSUE_WITH_EMPTY_DESCRIPTION.milestone) : Milestone.DefaultMilestone
+      );
+      expect(issue.state).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.state);
+      expect(issue.stateReason).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.stateReason);
+      if (ISSUE_WITH_EMPTY_DESCRIPTION.issueOrPr) {
+        expect(issue.issueOrPr).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.issueOrPr);
+      }
+      expect(issue.author).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.user.login);
+      expect(issue.isDraft).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.isDraft);
+      expect(issue.assignees).toEqual(ISSUE_WITH_EMPTY_DESCRIPTION.assignees.map((assignee) => assignee.login));
```

Comment:
You should test this using an issue with assignees. Also, your expected value should be an array of the assignees' logins, rather than using a method to generate it. 

Same thing when testing labels in the line below.